### PR TITLE
Fixes #14563: Add a generic method call to test generic method

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/14563

### CODE REVIEW

File: tests/acceptance/default_ncf.cf.sub [hunk 1]
```diff
@@ -80,3 +80,72 @@ bundle agent execute_serverspec(test_name, class_name, comment)
       "${report_string}";
 
 }
+
+bundle agent apply_gm(name, args, status, result_class)
```
Comment (amousset) [inline tests/acceptance/default_ncf.cf.sub]: Needs an explanation

**Comment by Fdall** [comment]
Commit modified

**Comment by amousset** [comment]
This is great!

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**